### PR TITLE
Markdown audit part 3

### PR DESCRIPTION
**Pull request by aw-asana**
Audits and reorganizes the (current) sections of documentation from "Developer Sandbox" to "Official Client Libraries"

### CODE REVIEW

File: source/includes/markdown/_03-auth.html.md [hunk 1]
```diff
@@ -356,7 +356,7 @@ curl https://app.asana.com/api/1.0/users/me \
   -H "Authorization: Bearer ACCESS_TOKEN"
 ```
 
-You can generate a personal access token from the Asana [developer console](https://app.asana.com/0/developer-console). See the [Authentication quick start](/docs/authentication-quick-start) for detailed instructions on getting started with PATs. 
+You can generate a personal access token from the Asana [developer console](https://app.asana.com/0/developer-console). See the [authenticating](/docs/authenticating) for detailed instructions on getting started with PATs.
```
Comment (jv-asana) [inline source/includes/markdown/_03-auth.html.md:359]: This sentence doesn't quite flow "See the authenticating for detailed"

Comment (jv-asana) [inline source/includes/markdown/_03-auth.html.md:359]: Did you check if there are external resources linked to `/docs/authentication-quick-start`? This change will break external links to this resource

File: source/includes/markdown/_07-pagination.html.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ curl "https://app.asana.com/api/1.0/tasks?project=1337&limit=5&offset=eyJ0eXAiOJ
 ```
 
 Note that all of Asana's official [client
-libraries](/docs/official-client-libraries) support
+libraries](/docs/client-libraries) support
```
Comment (jv-asana) [inline source/includes/markdown/_07-pagination.html.md:57]: Same idea here. Are we linking to this resource anywhere externally?

File: source/includes/markdown/_03-auth.html.md (hunk 1 above)
Comment (aw-asana) [reply source/includes/markdown/_03-auth.html.md:359]: Fixed!

File: source/includes/markdown/_07-pagination.html.md (hunk 2 above)
Comment (aw-asana) [reply source/includes/markdown/_07-pagination.html.md:57]: Fixed!

File: source/includes/markdown/_03-auth.html.md (hunk 1 above)
Comment (aw-asana) [reply source/includes/markdown/_03-auth.html.md]: Fixed!

**Review by jv-asana — APPROVED** [review]
LGTM

**Pull request merged**